lib/src/components/NxButton/__tests__/NxButton.test.tsx

```diff
@@ -5,82 +5,68 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
-import { mount, shallow } from 'enzyme';
-import { faCheck, faTrashAlt } from '@fortawesome/free-solid-svg-icons';
+import { waitFor, render } from '@testing-library/react';
+import { rtlRender, rtlRenderElement } from '../../../__testutils__/rtlUtils';
 
-import NxFontAwesomeIcon from '../../NxFontAwesomeIcon/NxFontAwesomeIcon';
 import NxButton from '../NxButton';
-import { TooltipContext } from '../../NxTooltip/NxTooltip';
 
 describe('NxButton', function() {
-  it('renders a button', function() {
-    const button = shallow(<NxButton />);
+  const quickRender = rtlRender(NxButton, {}),
+      renderEl = rtlRenderElement(NxButton, {});
 
-    expect(button).toHaveClassName('nx-btn');
+  it('renders a button', function() {
+    expect(quickRender().getByRole('button').tagName).toBe('BUTTON');
```

Comment:
In this case I think this is an implementation detail that doesn't need to be tested

Reply:
Removed here 86e0e240ca7e34829756a1c515c2ed3d645221dd.